tests/core/test_model.py

```diff
@@ -1320,6 +1321,93 @@ def model2_entrypoint(evaluator: MacroEvaluator) -> str:
     )
 
 
+def test_reference_model_in_python_context(tmp_path, assert_exp_eq) -> None:
```

Comment:
it's more robust and easier to maintain if this is just in sushi, can you add an example there?

Reply:
yea but as i said , testing in sushi would also be more robust, can you put a simpl eversion in sushi?

Reply:
So keep this and also update sushi with a simple example of this functionality?